cypress_test/integration_tests/desktop/writer/invalidations_spec.js

```diff
@@ -20,6 +20,7 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Invalidation tests.', func
 		helper.setupAndLoadDocument('writer/invalidations.odt');
 		desktopHelper.switchUIToNotebookbar();
 		cy.cGet('div.clipboard').as('clipboard');
+		cy.wait(3000);
```

Comment:
Nit: waiting on a specific condition would be better, if you can find a way to do that.